ocrd_network/ocrd_network/logging.py

```diff
@@ -0,0 +1,48 @@
+from os import makedirs
+from os.path import join
+from ocrd_utils import safe_filename
+
+OCRD_NETWORK_MODULES = [
+    "mets_servers",
+    "processing_jobs",
+    "processing_servers",
+    "processing_workers",
+    "processor_servers"
+]
+
+
+def get_root_logging_dir(module_name: str) -> str:
+    if module_name not in OCRD_NETWORK_MODULES:
+        raise ValueError(f"Invalid module name: {module_name}, should be one of: {OCRD_NETWORK_MODULES}")
+    # TODO: Utilize env variable to set the root
+    module_log_dir = join("/tmp/ocrd_network_logs", module_name)
+    makedirs(name=module_log_dir, exist_ok=True)
+    return module_log_dir
+
+
+def get_cache_locked_pages_logging_file_path() -> str:
+    return join(get_root_logging_dir("processing_servers"), f"cache_locked_pages.log")
```

Comment:
```suggestion
    return get_root_logging_dir("processing_servers") / "cache_locked_pages.log"
```

Also for the other calls.

Reply:
This does the right concatenation by platform, also Windows. The `/` operator is just overloaded to allow writing `Path('a') / 'b' instead of `Path('a', 'b'), it is semantically equivalent to `join('a', 'b')` and under the hood, it uses `os.join` IIRC. cf https://docs.python.org/3/library/pathlib.html#operators

Reply:
Did not know that. Thanks!